pkg/scheduler/BUILD

```diff
@@ -34,7 +34,6 @@ go_library(
         "//staging/src/k8s.io/apimachinery/pkg/apis/meta/v1:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/fields:go_default_library",
         "//staging/src/k8s.io/apimachinery/pkg/runtime:go_default_library",
-        "//staging/src/k8s.io/apimachinery/pkg/types:go_default_library",
```

Comment:
/hold

removing this is failing batch builds, see https://prow.k8s.io/?repo=kubernetes%2Fkubernetes&type=batch&pull=master%2C90544%2C90660%2C90861%2C91045%2C91082%2C91083%2C91084%2C91085%2C91099%2C91106

Reply:
Sry for that. Updated.